model/src/main/resources/reference.yml

```diff
@@ -193,7 +193,11 @@ ort:
     options:
       # A map of maps from scanner class names to scanner-specific key-value pairs.
       ScanCode:
+        # Command line options that affect the ScanCode output. If changed, stored scan results that were created with
+        # different options are not reused.
         commandLine: '--copyright --license --info --strip-root --timeout 300'
+        # Command line options that do not affect the ScanCode output.
```

Comment:
Nit: Should blank lines for grouping be added to make more clear that the comment does not refer to `parseLicenseExpressions` also?